content/guides/mobile/ios/quick-start/2-install-ios-sdk.md

```diff
@@ -0,0 +1,117 @@
+---
+type: quick-start
+hide_in_page_nav: true
+---
+
+# Install the iOS SDK
+
+With an iOS application in place, you will need to import the required **Box
+iOS SDK** dependencies into your project using one of the available package
+manager options.
+
+## Select a package manager to use
+
+<Grid columns='3'>
+  <Choose option='ios.pm_type' value='carthage' color='blue'>
+    # Carthage
+
+    Carthage is a decentralized dependency manager for Swift
+    and Objective-C Cocoa projects. It is open-source and built
+    with Swift.
+  </Choose>
+
+  <Choose option='ios.pm_type' value='cocoapods' color='none'>
+    # CocoaPods
+
+    CocoaPods is a centralized dependency manager for Swift and
+    Objective-C Cocoa projects. It is open-source and was built
+    with Ruby.
+  </Choose>
+
+  <Choose option='ios.pm_type' value='swift' color='blue'>
+    # Swift Package Manager
+
+    Included with Swift since version 3.0, the Swift Package
+    Manager is a tool for managing the distribution of source
+    code, aimed at making it easy to share your code and reuse
+    others’ code.
+  </Choose>
+</Grid>
+
+<Choice option='ios.pm_type' value='carthage' color='blue'>
+  # Install the iOS SDK using Carthage
+
+  1. From a terminal window, install Carthage: `brew install carthage`.
+    [See here](https://github.com/Carthage/Carthage#installing-carthage)
+    for other installation methods.
+  2. At the root of your iOS application folder, where the `{APPNAME}.xcodeproj`
+     is located, create a new file named `Cartfile`, without an extension.
+  3. Load `Cartfile`, add the **Box iOS SDK** dependency with
+     `git "https://github.com/box/box-ios-sdk.git" ~> 3.0`, then save and close
+     the file.
+  4. From the terminal, in the folder where the `Cartfile` is present, update
+     all dependencies: `carthage update --platform iOS`. A `Cartfile.resolved`
+     file and a `Carthage` directory will be created in the directory. 
+  5. Within a Finder or File Explorer window load the `Carthage` -> `Build` ->
+     `iOS` directory. This should have a framework file named
+     `BoxSDK.framework`. Keep this window open as we'll use it to add the
+     framework to our project in the next step.
+  6. In your Xcode project, click on the name of your application at the top of
+     the project navigator on the left. In the content that displays, click on
+     the name of your project under the **TARGETS** option. Scroll down to
+     **Frameworks, Libraries, and Embedded Content**. 
+  7. Drag `BoxSDK.framework` from our Finder window over to the frameworks
+     section.
+    <ImageFrame border center>
+      ![Add framework to project](./framework-carthage-add.gif)
```

Comment:
Not sure why this image isn't loading. My suspicion is it's related to the bad rendering of the block.